openapi.yml

```diff
@@ -7127,28 +7114,6 @@ components:
           items:
             $ref: '#/components/schemas/Tracker'
           readOnly: true
-        meta_attr:
-          type: object
-          properties:
-            affectedness:
-              type: string
-            component:
-              type: string
-            impact:
-              type: string
-            module_name:
-              type: string
-            module_stream:
-              type: string
-            ps_component:
-              type: string
-            ps_module:
-              type: string
-            ps_product:
-              type: string
-            resolution:
-              type: string
-          readOnly: true
```

Comment:
With the upgrade of `drf-spectacular` (which is required due to a breaking change in the upgrade of drf) these disappeared for only some of the serializers. In trying to figure out why and how the schema is generated, it appears the issue is that to generate the schema it performs some requests to the various endpoints but with no query params, so it does not include the `include_meta_attr` params and the field gets removed in `IncludeMetaAttrMixin`.

Why this happened only for some serializes (for instance, it happened for `AffectSerializer` but not for `AffectPostSerializer` which just inherits from the former, even though in this case it shouldn't even have them since it's read-only) is beyond me. But this was also happening before for the `history` field on all serializers, since the requests do not include the `include_history` param either, which makes me wonder if these should even be here to begin with. If we do, we might want to show the `history` field as well, but this will require some extensive work to understand how the schema is generating these fields that can be dynamically removed in the serializer.

Reply:
I am thinking how to understand this change from the versioning point of view. The API does not get changed, only the documentation, correct?

Reply:
Correct, the API does not change and the `meta_attr` are still returned only if you pass the `include_meta_attr` param. But the documentation stopped showing this param, which is probably due to a change in how the documentation is generated in the new version.

Reply:
Maybe that is actually for good as those meta attributes have no insurance of even existence but they were already being used as production data multiple times.